services/libs/common/src/i18n/en.ts

```diff
@@ -152,6 +152,12 @@ const en = {
     git: {
       noIntegration: 'The Git integration is not configured.',
     },
+    jira: {
+      invalidUrl:
```

Comment:
We dont use translations anymore so in further prs you can also include directly in code.